case/src/main/kotlin/com/ritense/case/web/rest/CaseTabResource.kt

```diff
@@ -42,4 +44,13 @@ class CaseTabResource(
         return ResponseEntity.ok(tabs)
     }
 
+    @GetMapping("/v1/document/{documentId}/tab")
```

Comment:
We should probably deprecate the old one

Reply:
Done